engine/battle/ai/scoring.asm

```diff
@@ -736,9 +766,9 @@ AI_Smart_MirrorMove:
 	ret nc
 
 ; ...or dismiss this move if enemy is faster than player.
-	jp AIDiscourageMove
+	jp AIDismissMove
 
-; If the player did use a move last turn...
+; If the player did use a useful move last turn...
```

Comment:
What counts as useful?

Reply:
A move in UsefulMoves.

Reply:
I tend to use adjectives that match the name of some data structures. Otherwise i don't really use adjectives. `useful` will mostly if not always refer to UsefulMoves, `reckless` will refer to RecklessMoves, etc.

Reply:
Would the following wording allowed ? `If the player did use a move in UsefulMoves last turn...`

Reply:
I'm not qualified to say, but that would make it clearer.